### ISSUE TITLE
Call on shoot when using EffShoot

### ISSUE THREAD

**Issue by TheLimeGlass** [SkriptLang/Skript#4969]
### Suggestion

When a Skripter uses the EffShoot effect, it should call the on shoot event.

**Comment by UnderscoreTud** [comment]
It already does that
```vb
on right click:
    make player shoot arrow

on shoot:
    broadcast "SHOT"
```

![image](https://user-images.githubusercontent.com/98935832/181024238-c0b424c5-b5ea-4e1b-9432-4f7b08fba9aa.png)

**Comment by TheLimeGlass** [comment]
Could not replicate when I attempted it.

**Comment by AyhamAl-Ali** [comment]
It actually exists
![image](https://user-images.githubusercontent.com/20037329/181087508-934a5aaf-0b20-41c4-81be-6f9913483770.png)


see 
https://github.com/SkriptLang/Skript/blob/e9d04e7604c13d0de8d210f88ea0f021cdace009/src/main/java/ch/njol/skript/effects/EffShoot.java#L99
and ProjectileLaunchEvent extends EntitySpawnEvent see
https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/entity/ProjectileLaunchEvent.html

**Comment by TheLimeGlass** [comment]
Try it without an arrow. Try a custom entity.

**Comment by AyhamAl-Ali** [comment]
Ah yea, that doesn't trigger the shoot event indeed.

Support should be added here 
https://github.com/SkriptLang/Skript/blob/e9d04e7604c13d0de8d210f88ea0f021cdace009/src/main/java/ch/njol/skript/effects/EffShoot.java#L109-L115

**Comment by TPGamesNL** [comment]
I don't think this should be added: `on shoot`, or ProjectileLaunchEvent only accepts projectiles [[1]](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/entity/ProjectileLaunchEvent.html#getEntity()) (technically the constructor doesn't, but getEntity does which will cause issues if we call this event). The name of the Bukkit event also gives it away.
